Index: smokeconfig.xml
===================================================================
--- smokeconfig.xml	(revision 1055285)
+++ smokeconfig.xml	(working copy)
@@ -1,7 +1,9 @@
 <config>
     <moduleName>qtwebkit</moduleName>
     <parentModules>
-        <module>qt</module>
+        <module>qtcore</module>
+        <module>qtnetwork</module>
+        <module>qtgui</module>
     </parentModules>
     <parts>1</parts>
     <scalarTypes>
Index: CMakeLists.txt
===================================================================
--- CMakeLists.txt	(revision 1055285)
+++ CMakeLists.txt	(working copy)
@@ -22,7 +22,14 @@
 ENDIF(CMAKE_CXX_FLAGS MATCHES "-fvisibility")
 add_library(smokeqtwebkit SHARED ${smokeqtwebkit_LIB_SRCS})
 
-target_link_libraries(smokeqtwebkit smokeqt ${QT_QTCORE_LIBRARY} ${QT_QTGUI_LIBRARY} ${QT_QTNETWORK_LIBRARY} ${QT_QTWEBKIT_LIBRARY})
+target_link_libraries(  smokeqtwebkit 
+                        smokeqtcore 
+                        smokeqtgui 
+                        smokeqtnetwork 
+                        ${QT_QTCORE_LIBRARY} 
+                        ${QT_QTGUI_LIBRARY} 
+                        ${QT_QTNETWORK_LIBRARY} 
+                        ${QT_QTWEBKIT_LIBRARY} )
 
 set_target_properties(smokeqtwebkit PROPERTIES VERSION 2.1.0 SOVERSION 3 )
 
